README.md

```diff
@@ -4,30 +4,43 @@
   </a>
 </p>
 <p align="center">
-    <a href="https://datapane.com">Datapane Teams</a> |
-    <a href="https://docs.datapane.com">Documentation</a> |
-    <a href="https://datapane.github.io/datapane/">API Docs</a> |
-    <a href="https://docs.datapane.com/changelog">Changelog</a> |
-    <a href="https://twitter.com/datapaneapp">Twitter</a> |
-    <a href="https://blog.datapane.com">Blog</a>
-    <br /><br />
-    <a href="https://pypi.org/project/datapane/">
-        <img src="https://img.shields.io/pypi/dm/datapane?label=pip%20downloads" alt="Pip Downloads" />
-    </a>
-    <a href="https://pypi.org/project/datapane/">
-        <img src="https://img.shields.io/pypi/v/datapane?color=blue" alt="Latest release" />
-    </a>
-    <a href="https://anaconda.org/conda-forge/datapane">
-        <img alt="Conda (channel only)" src="https://img.shields.io/conda/vn/conda-forge/datapane">
-    </a>
+  <a href="https://datapane.com">Datapane.com</a> |
+  <a href="https://docs.datapane.com">Docs</a> |
+  <a href="https://twitter.com/datapaneapp">Twitter</a> |
+  <a href="https://forum.datapane.com">Forums</a>   | <a href="https://chat.datapane.com">Discord</a>
+</p>
+<p align='center'>
+  <a href="https://pypi.org/project/datapane/">
+      <img src="https://img.shields.io/pypi/dm/datapane?label=pip%20downloads" alt="Pip Downloads" />
+  </a>
+  <a href="https://pypi.org/project/datapane/">
+      <img src="https://img.shields.io/pypi/v/datapane?color=blue" alt="Latest release" />
+  </a>
+  <a href="https://anaconda.org/conda-forge/datapane">
+      <img alt="Conda (channel only)" src="https://img.shields.io/conda/vn/conda-forge/datapane">
+  </a>
 </p>
-<h4>Share interactive plots and data in 3 lines of Python.</h4>
 
-Datapane is a Python library for building interactive reports for your end-users in seconds.
+<p align='center'>
+  <h1 align='center'> Build interactive reports in 3 lines of Python</h1>
+</p>
+Datapane is an open-source framework makes it easy to create beautiful reports from anywhere you can run Python.
+Love analyzing data in Python but struggle to share results and insights? Datapane is for you!
+<br>
+<br>
+<br>
+
+<p align="center">
+  <a href="https://datapane.com">
+    <img src="https://user-images.githubusercontent.com/3541695/176245723-3874bf39-b2c3-42e0-802c-6d8176e7b7d2.png"  width='75%'>
```

Comment:
This image is front and centre – shall we make a better one?

Reply:
yeah do we have a better one? alternatively, we can have this as a placeholder and once the video is done we could put that there.

Reply:
Just put this together - what do you think?

<img width="1288" alt="new_dp_promo" src="https://user-images.githubusercontent.com/15690380/176505445-e47da93b-237f-4a1d-b316-9de1cdcdc090.png">